centos7.sh

```diff
@@ -31,45 +31,60 @@ msg_opt_complete(){
 }
 
 do_install(){
+
+  # practical options for curl. use with care.
+  CURL_OPTS="--insecure"
   # install basic packages
   yum install -y yum-utils device-mapper-persistent-data lvm2
 
-  # install docker-ce and EPEL repos
+  # install docker-ce
   yum-config-manager --add-repo https://download.docker.com/linux/centos/docker-ce.repo
-  yum install -y epel-release
+  # install EPEL using rpm so it works for both RHEL and CentOS
+  yum install -y http://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
```

Comment:
Same comment as below .. not a fan of direct installing static rpm files although I accept the version of this one is attempting to be a static target.

Reply:
Me neither, it's just what it takes if you want this to support RHEL